### DISCUSSION TITLE
Can't build because ANDROID_HOME is not found?

### DISCUSSION THREAD

**Discussion by tonywei92** [subosito/flutter-action#275] [General]
And I didn't see Android SDK download either? maybe i'm missing something?

**Comment by koji-1009** [comment]
Which version of machine do you use?

From the documentation, if it is ubuntu-latest, ubuntu-22.04, or ubuntu-20.04, it seems to be set.
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md#android
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#android

**Comment by bartekpacia** [comment]
GitHub provides Android SDK in their pre-built images. If you do something differently, you have to obtain Android SDK yourself.
